xcodeproj/internal/bazel_integration_files/generate_bazel_dependencies.sh

```diff
@@ -108,7 +110,17 @@ fi
 # Build
 
 build_pre_config_flags=(
-  "--experimental_remote_download_regex=.*\.indexstore/.*|.*\.a$|.*\.swiftdoc$|.*\.swiftmodule$|.*\.swiftsourceinfo$|.*\.swift$"
+  # Include the following:
+  #
+  # - .indexstore directories to allow importing indexes
+  # - .a and .swiftmodule files for lldb
+  # - primary compilation input files (.c, .C, .cc, .cl, .cpp, .cu, .cxx, .c++,
+  #   .m, .mm, .swift) for Xcode build input checking
+  #
+  # This is brittle. If different file extensions are used for compilation
+  # inputs, they will need to be added to this list. Ideally we can stop doing
+  # this once Bazel adds support for a Remote Output Service.
+  "--experimental_remote_download_regex=.*\.indexstore/.*|.*\.(a|c|C|cc|cl|cpp|cu|cxx|c++|m|mm|swift|swiftmodule)$"
```

Comment:
Was it intentional to drop `swiftdoc` and `swiftsourceinfo` here?

Reply:
Yes. They are only needed for indexing, and that's covered in the other `--experimental_remote_download_regex`.

Now that you ask though, I'm not sure which location Xcode uses. It might/is probably the main one instead of the index output base. I'll add them back.